docs/scanners/uswds.md

```diff
@@ -1,26 +1,26 @@
 # U.S. Web Design System Scanner
 
 ## Summary
-The [USWDS scanner](https://site-scanning.app.cloud.gov/searchUSWDS/) checks each domain for the use of [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/) code and the code version.
+The [U.S. Web Design System (USWDS) scanner](https://site-scanning.app.cloud.gov/searchUSWDS/) checks each domain for the use of [USWDS](https://designsystem.digital.gov/) code and for the USWDS code version.
 
 Specifically, the scan [searches for](https://github.com/18F/domain-scan/blob/tspencer/200scanner/scanners/uswds2.py#L36-L123): 
 
-* Presence of `-usa` class in CSS
+* `usa-` prefixed classes in HTML
 * `uswds` in HTML
 * `.usa-` in HTML
```

Comment:
@timothy-spencer Can you take a look at this? If the scanner is searching for `.usa-` it might not be getting accurate hits, since the period `.` wouldn't be included in an html class name. Like `<button class="usa-button usa-button--secondary">`)

And in this search instance the `.` is actually escaped out (`\.`) and not meant to be a regex wildcard.

see: https://github.com/18F/domain-scan/blob/8223c7110d5f2bfbb8205379e09b4cea11519983/scanners/uswds2.py#L36

Reply:
The scanner is searching for two different usa regexes:
- `class.*"usa-`
- `\.usa-`

The first would match `<button class="usa-button usa-button--secondary">`, but the second one would not.  It maps to the `usa_classes_detected` analysis number.

The second one from https://github.com/18F/domain-scan/blob/8223c7110d5f2bfbb8205379e09b4cea11519983/scanners/uswds2.py#L53 would match anything with `.usa-` in the text.  I think I put that in there as a sort of catchall, and maps to the `usa_detected` analysis number, but maybe it's not doing anything at all?

Reply:
To go back to the original text, it is accurate.  Would you like something different?